windtunnel/windtunnel_outputs.py

```diff
@@ -0,0 +1,153 @@
+"""
+Handles loading force coefficients, generating streamlines, and interpolating
+pressure fields over simulation meshes.
+"""
+
+import os
+import pathlib
+
+import numpy as np
+import pyvista as pv
+
+
+class WindTunnelOutputs:
+    """
+    Manages WindTunnel simulation outputs.
+
+    Provides methods to load results, including force coefficients, streamlines,
+    and pressure fields, from a WindTunnel simulation output directory.
+
+    Attributes:
+        simulation_path (str): Path to simulation output directory.
+        time_steps (int): Number of time steps in the simulation.
+        domain_mesh (pv.PolyData): Domain mesh from the simulation.
+        object_mesh (pv.PolyData): Object mesh from the simulation.
+    """
+
+    def __init__(self, simulation_path: str):
+        """
+        Initializes WindTunnelOutputs with the simulation directory.
+
+        Args:
+            simulation_path (str): Path to simulation output directory.
+        """
+        self.simulation_path = simulation_path
+        self.time_steps = self._get_num_time_steps()
+        self.domain_mesh, self.object_mesh = self._get_output_meshes()
+
+    def get_force_coefficients(self):
+        """
+        Loads force coefficients from the simulation output.
+
+        Returns:
+            dict: Force coefficients including 'Moment', 'Drag', 'Lift',
+            'Front Lift', and 'Rear Lift'.
+        """
+        force_coefficients_path = os.path.join(self.simulation_path,
+                                               "postProcessing", "forceCoeffs1",
+                                               "0", "forceCoeffs.dat")
+        coefficients = np.loadtxt(force_coefficients_path)[-1]
+        coefficients_labels = [
+            "Moment", "Drag", "Lift", "Front Lift", "Rear Lift"
+        ]
+        coefficients_dict = dict(
+            zip(coefficients_labels, map(float, coefficients[1:])))
+
+        return coefficients_dict
+
+    def get_streamlines(self,
+                        n_points=100,
+                        initial_step_length=1,
+                        source_radius=1.1,
+                        source_center=(0, 0, 0),
+                        streamline_radius=0.005):
+        """
+        Generates streamlines from the domain mesh.
+
+        Args:
+            n_points (int, optional): Number of seed points. Defaults to 100.
```

Comment:
"This will also be the number of streamlines in the final output"

Reply:
```suggestion
            n_points (int, optional): Number of streamlines. Defaults to 100.
```